Dungeons and Dieting/Dungeons and Dieting/InventoryViewController.swift

```diff
@@ -242,49 +287,49 @@ class InventoryViewController:UIViewController
+        inventoryUIViewer19 = UIImageView(image: inventoryUIimage)
      
         
-        inventoryUIViewer.frame = CGRect(x: 50, y: 500, width: 60, height: 60)
-         inventoryUIViewer2.frame = CGRect(x: 100, y: 500, width: 60, height: 60)
-         inventoryUIViewer3.frame = CGRect(x: 150, y: 500, width: 60, height: 60)
-         inventoryUIViewer4.frame = CGRect(x: 200, y: 500, width: 60, height: 60)
-         inventoryUIViewer5.frame = CGRect(x: 250, y: 500, width: 60, height: 60)
-         inventoryUIViewer6.frame = CGRect(x: 300, y: 500, width: 60, height: 60)
+        inventoryUIViewer.frame = CGRect(x: 50, y: 550, width: 60, height: 60)
+        inventoryUIViewer2.frame = CGRect(x: 100, y: 550, width: 60, height: 60)
+        inventoryUIViewer3.frame = CGRect(x: 150, y: 550, width: 60, height: 60)
+        inventoryUIViewer4.frame = CGRect(x: 200, y: 550, width: 60, height: 60)
+        inventoryUIViewer5.frame = CGRect(x: 250, y: 550, width: 60, height: 60)
+        inventoryUIViewer6.frame = CGRect(x: 300, y: 550, width: 60, height: 60)
        
-        inventoryUIViewer7.frame = CGRect(x: 50, y: 550, width: 60, height: 60)
-        inventoryUIViewer8.frame = CGRect(x: 100, y: 550, width: 60, height: 60)
-        inventoryUIViewer9.frame = CGRect(x: 150, y: 550, width: 60, height: 60)
-         inventoryUIViewer10.frame = CGRect(x:200, y: 550, width: 60, height: 60)
-        inventoryUIViewer11.frame = CGRect(x: 200, y: 550, width: 60, height: 60)
-        inventoryUIViewer12.frame = CGRect(x: 250, y: 550, width: 60, height: 60)
-        inventoryUIViewer13.frame = CGRect(x: 300, y: 550, width: 60, height: 60)
+        inventoryUIViewer7.frame = CGRect(x: 50, y: 600, width: 60, height: 60)
+        inventoryUIViewer8.frame = CGRect(x: 100, y: 600, width: 60, height: 60)
+        inventoryUIViewer9.frame = CGRect(x: 150, y: 600, width: 60, height: 60)
+         inventoryUIViewer10.frame = CGRect(x:200, y: 600, width: 60, height: 60)
+        inventoryUIViewer11.frame = CGRect(x: 200, y: 600, width: 60, height: 60)
+        inventoryUIViewer12.frame = CGRect(x: 250, y: 600, width: 60, height: 60)
+        inventoryUIViewer13.frame = CGRect(x: 300, y: 600, width: 60, height: 60)
      
-        inventoryUIViewer14.frame = CGRect(x: 50, y: 600, width: 60, height: 60)
-      inventoryUIViewer15.frame = CGRect(x: 100, y: 600, width: 60, height: 60)
-      inventoryUIViewer16.frame = CGRect(x: 150, y: 600, width: 60, height: 60)
-      inventoryUIViewer17.frame = CGRect(x: 200, y: 600, width: 60, height: 60)
-      inventoryUIViewer18.frame = CGRect(x: 250, y: 600, width: 60, height: 60)
-      inventoryUIViewer19.frame = CGRect(x: 300, y: 600, width: 60, height: 60)
-        
+        inventoryUIViewer14.frame = CGRect(x: 50, y: 650, width: 60, height: 60)
+        inventoryUIViewer15.frame = CGRect(x: 100, y: 650, width: 60, height: 60)
+        inventoryUIViewer16.frame = CGRect(x: 150, y: 650, width: 60, height: 60)
```

Comment:
think about using layout anchors and constraints for consistent layout on different devices.